doc/sphinx/proof-engine/tactics.rst

```diff
@@ -202,28 +196,25 @@ For a goal :g:`A \/ B`, use :tacn:`left` to replace the current goal with :g:`A`
   of :n:`let-ins`, the patterns are applied to the arguments and :n:`let-in` variables.
 
 * :n:`( {+& @simple_intropattern} )` - matches a term that consists of one or more
-  right-hand nested binary inductive types in a form such as :g:`(a1 OP (a2 OP ...))`.
+  nested binary inductive types in a right-associative form such as :g:`(a1 OP (a2 OP ...))`.
   The splitting pattern can have more than 2 names, for example :g:`(H1 & H2 & H3)`
   matches :g:`A /\ B /\ C`.
   The inductive types must have a
   :ref:`single constructor with two parameters <intropattern_cons_note>`.
   :ref:`Example <intropattern_ampersand_ex>`
 
-* :n:`[ {+| @intropatterns} ]` - splits an inductive type that has
+* :n:`[ {*| @intropatterns} ]` - splits an inductive type that has
   :ref:`multiple constructors <intropattern_cons_note>`
   such as :n:`A \/ B`
-  into multiple subgoals.  The number of ``intropatterns`` must be the same as the number of
+  into multiple subgoals.  The number of :token:`intropatterns` must be the same as the number of
   constructors for the matched part.
 * :n:`[ {* @intropattern} ]` - splits an inductive type that has a
   :ref:`single constructor with multiple parameters <intropattern_cons_note>`
   such as :n:`A /\ B` into multiple hypotheses.  Use :n:`[H1 [H2 H3]]` to match :g:`A /\ B /\ C`.
-* :n:`[]` - splits an inductive type:  If the inductive type has multiple constructors,
+* :n:`[]` - splits an inductive type (a special case of the last 2 bullets):  If the inductive
```

Comment:
On the contrary, I would not say that it is a "special case". It has a peculiar semantics...